Unaccountable-Magpie/BangazonSite
# View Recent products on the home page

### Purpose:

Users can view the most recent products on the home page


### How it fits in context of the project:

Fulfills ticket #4  

### Specific Feature affected:

Home Index.cs
HomeController.cs
(also deleted the other home views because they had no purpose)

### How to test (Be thorough!):

1. Check into my branch "bs-home"

2. Run the program by clicking the IIExpress play button

3. Check to see that the build completes and three products show up on the home page

4. Click on them and make sure they route to their details

5. Run Dotnet Test in the terminal

6. There should be 4 passes and 0 fails
